source/shared/net/desktop-wallet/accounts-desktop.rst

```diff
@@ -57,7 +59,7 @@ When you select **Account list** at the top of the window, the button name toggl
 
 -  **Update credentials**: Add new credentials to the account. See :ref:`Add credentials to an account <multi-credentials>`.
 
--  **Register data**:
+-  **Register data**: Add a small message to the chain, such as a hash. Note that adding this message incurs a transaction fee.
```

Comment:
Do we need a disclaimer that data is public and cannot be removed? Or is that implicit